main.py

```
@@ -0,0 +1,479 @@
+            "page": page,
+            "limit": limit,
+            "total": 100
+        }
+    }
+
+
+# 좋아요 상태 확인
+@app.get("/posts/{post_id}/likes", response_model=post.PostLikeStatus)
+async def get_post_likes(
+        post_id: Annotated[str, Path(description="게시글 ID")],
+        authorization: Annotated[str, Header(description="로그인 토큰")]
```

Ruff (0.14.13)

Unused function argument: `authorization`

(ARG001)